tests/modules/programs/yt-dlp/yt-dlp-simple-config.nix

```
@@ -0,0 +1,24 @@
+{ ... }:
+
+{
+  programs.yt-dlp = {
+    enable = true;
+    settings = {
+      embed-thumbnail = true;
+      embed-subs = true;
```

Try having one boolean option with a `false` value:

```suggestion
      embed-subs = false;
```